### PR TITLE
Add ahrthr@ to reviewers

### PR DESCRIPTION
**Pull request by serathius**
Per discussion with existing maintainers, I nominate @ahrtr for the reviewer role.

@ahrtr thank you for your outstanding contributions to the project.

/cc @ptabor @gyuho @hexfusion @jpbetz @xiang90 @mitake @jingyih @wenjiaswe

### CODE REVIEW

**Review by spzala — APPROVED** [review]
Big +1
Thank you for your contributions @ahrtr

**Comment by ahrtr** [comment]
Thanks @serathius and all. 

Could you please change my email to `wachao@vmware.com` ?  cc @serathius

**Review by ptabor — APPROVED** [review]

File: MAINTAINERS [hunk 1]
```diff
@@ -26,4 +26,4 @@ Tobias Grieger <tobias.schottdorf@gmail.com> (@tbg) pkg:go.etcd.io/etcd/raft
 # REVIEWERS
 Lili Cosic <cosiclili@gmail.com> (lilic@) pkg:*
 Wilson Wang <wilson.wang@bytedance.com> (wilsonwang371@) pkg:*
-
+Benjamin <nkty_coolnick@163.com> (ahrtr@) pkg:*
```
Comment (serathius) [inline MAINTAINERS:29]:
```suggestion
Benjamin Wang <wachao@vmware.com> (ahrtr@) pkg:*
```

**Comment by serathius** [comment]
Done @[ahrtr](https://github.com/ahrtr)

**Review by gyuho — APPROVED** [review]

**Comment by ptabor** [comment]
Thank you for the contributions !

**Pull request merged**